File: javaScript/counter/counter.js
function counter(string) {
    let sum = 0;

    for (let i = 0; i < string.length; i++) {
        sum += Number(string[i]);
    }

    return {
        zero: string.length - sum,
        one: sum
    }
}

const string = "010010";
console.log(counter(string));

File: counter/counter.cpp
#include <iostream>
#include <string>

void counter(std::string str) {
	unsigned int sum = 0;
	int n = std::stoi(str);

	while (n != 0) {
		sum += n % 10;
		n /= 10;
	}

	std::cout << "The number of 1s: " << sum << " \nThe number of 0s: " << str.length() - sum << " \n";
}

int main() {
	std::string str = "010001";
	counter(str);

	return 0;
}